src/elements/ValueList.vue

```diff
@@ -19,7 +18,7 @@
           level="h5"
           font-weight="light"
           line-height="compact">
-          {{ items.value }}
+          <div v-html="items.value" />
```

Comment:
@sgashler I think you are on the right track, but per HTML standards, you can not put div inside heading elements such as <h1> - <h6>.  You'll need to add a span or add the 'v-html' directly to the heading.